docs/guides/getting-started-java-spring.md

```diff
@@ -181,7 +181,24 @@ In your terminal, run `mvn spring-boot:run`, where you should see the `charging
 
 To start a process instance programmatically, take the following steps:
 
-1. In `ProcessPaymentsApplication.java`, convert the application to a `CommandLineRunner`, by adding `implements CommandLineRunner` to the `ProcessPaymentsApplication` class declaration. Instantiate a static `Logger` variable, and an instance variable named `zeebeClient` with the `@Autowired` annotation.
+1. In `ProcessPaymentsApplication.java`, add the following dependencies after the package definition:
+
+```java
+package io.camunda.demo.process_payments;
+
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.spring.client.annotation.Deployment;
+```
```

Comment:
I lost an embarrassing amount of time trying to track down the most basic of imports (`java.util.Map`). We specify the imports in the previous code-heavy step; specifying them here could save someone like me from import-hunting.